src/store/src/InMemoryStore.php

```diff
@@ -48,18 +48,18 @@ public function add(VectorDocument ...$documents): void
      */
     public function query(Vector $vector, array $options = [], ?float $minScore = null): array
     {
-        $strategy = match ($this->similarity) {
-            self::COSINE_SIMILARITY => $this->cosineSimilarity(...),
+        $distanceMetricStrategy = match ($this->distanceMetric) {
```

Comment:
```suggestion
        $strategy = match ($this->distanceMetric) {
```

no need to rename I guess